### DISCUSSION TITLE
How does one set the description of a tag?

### DISCUSSION THREAD

**Discussion by leeand00** [GothenburgBitFactory/timewarrior#636] [Q&A] [answered]
When I run `timew tags` it displays the available tags and also includes a description column; 

But it doesn't include anything in the description column; and I do not know how to fill in that text; is there a way to "set the description of a tag"?

I feel like maybe it has something to do with the :annotation hint, but I'm not sure.

To be clear this is the description I am referring to:

```
$ timew tags

Tag            Description
-------------- -----------
Prepare Slides -
Presentation   -
```

**Comment by lauft** [comment, marked as answer]
Currently, you have to set the tag description in the configuration file, e.g. when you have a tag `FOO` you can add a description with
```
timew config :yes tags.FOO.description "Lorem ipsum dolor sit amet"
```
Then you should get as output to `timew tags`:
```
$ timew tags

Tag Description
--- --------------------------
FOO Lorem ipsum dolor sit amet
```

However, there is a catch: Tags with spaces are not supported yet with this. 😬 
(The reason is that the space in the tag breaks the line handling in the config file).
